data-access-analysis-tools/adql-recipes.rst

```diff
@@ -0,0 +1,222 @@
+This parameter is ``True`` if a source has no children, is in the inner region of a coadd patch, is in the inner region of a coadd tract, and is not detected in a pseudo-filter.
+Including this constraint will remove any duplicates (i.e., will not include *both* a parent and its deblended children).
+
+Additional external resources for learning about SQL, ADQL, and Qserv include:
+ - `W3 School's SQL Tutorial <https://www.w3schools.com/sql/default.asp>`__
+ - `IVOA's ADQL Documentation <https://www.ivoa.net/documents/ADQL/20180112/PR-ADQL-2.1-20180112.html>`__
+ - `LSST Qserv User Guide <https://qserv.lsst.io/user/index.html>`__
+
+
+.. _Adql-Recipes-Explore-Tables:
+
+Exploring tables
+================
+
+When learning about the contents of a table, it can be handy to simply retrieve all columns for "a bunch" (hundreds to thousands) of rows
+and take a look at the results.
+For this use-case, it is recommended to use the ``SELECT TOP`` statement, like in the example below that just retrieves the first 100 rows of the ``Object`` table.
+
+.. code-block:: SQL
+    SELECT TOP 100 * FROM dp02_dc2_catalogs.Object
+
+
+
+.. _Adql-Recipes-Cone-Search:
+
+Cone Search
+===========
+
+Retrieve the ``coord_dec`` and ``coord_ra`` columns from the ``Object`` table for objects within a 0.05 degree radius of RA = 62, Dec = -37.
+
+.. code-block:: SQL
+
+   SELECT coord_dec, coord_ra 
+   FROM dp02_dc2_catalogs.Object 
+   WHERE CONTAINS(POINT('ICRS', coord_ra, coord_dec), 
+   CIRCLE('ICRS', 62, -37, 0.05)) = 1
+
+
+
+.. _Adql-Recipes-FluxToMags:
+
+Convert fluxes to magnitudes
+============================
+
+As above, retrieve the ``coord_dec`` and ``coord_ra`` columns from the ``Object`` table for objects within a 0.05 degree radius of RA = 62, Dec = -37,
+and also retrieve the g-band AB magnitude and magnitude error.
+The ``scisql`` functions used below can be applied to any flux column.
+
+.. code-block:: SQL
+
+   SELECT coord_dec, coord_ra, 
+   scisql_nanojanskyToAbMag(g_calibFlux) AS g_calibMag, 
+   scisql_nanojanskyToAbMagSigma(g_calibFlux, g_calibFluxErr) as g_calibMagErr 
+   FROM dp02_dc2_catalogs.Object 
+   WHERE CONTAINS(POINT('ICRS', coord_ra, coord_dec), 
+   CIRCLE('ICRS', 62, -37, 0.05)) = 1
+
+
+.. _Adql-Recipes-Table-Joins:
+
+Table joins
+===========
+
+Below, the Source and CcdVisit table are joined in order to obtain the date and seeing from the CcdVisit table.
+Any two tables can be joined so long as they have an index in common.
+
+This query also renames (nicknames) columns and tables using ``AS``,
+and applies a spatial constraint, a temporal constraint (using ``obsStartMJD``), 
+and constraints on the band, extendedness, and flux value.
+
+Additional external resources on SQL table joins:
+ - `W2 School's SQL tutorial: joins <https://www.w3schools.com/sql/sql_join.asp>`__
+ - `The Data School: SQL Joins Explained Visually <https://dataschool.com/how-to-teach-people-sql/sql-join-types-explained-visually/>`__
+
```

Comment:
ooooh. this is excellent material, I didn't know this was possible! Will try this next time.

Reply:
Super!